Formula/auditbeat.rb

```diff
@@ -39,7 +31,7 @@ def install
     ENV.prepend_create_path "PYTHONPATH", buildpath/"vendor/lib/python2.7/site-packages"
 
     resource("virtualenv").stage do
-      system "python", *Language::Python.setup_install_args(buildpath/"vendor")
+      system "python3", *Language::Python.setup_install_args(buildpath/"vendor")
```

Comment:
This might need to be 3.8 to avoid picking up the system one on catalina

Reply:
Oh right.

Reply:
Maybe I'm being overly cautious then.

Reply:
You should use: `system Formula["python@3.8"].opt_bin/"python3"`

Reply:
Ok, done for all `*beat` formulae. However there are still a lot of other formula in `homebrew-core`, which still just call `system "python3"`.